homeassistant/components/victron_ble/strings.json

```
@@ -0,0 +1,29 @@
+{
+  "config": {
+    "flow_title": "{title}",
+    "step": {
+      "user": {
+        "data": {
+          "address": "The Bluetooth address of the Victron device."
+        },
+        "data_description": {
+          "address": "This Bluetooth address is automatically discovered. You may view a device's Bluetooth address in the VictronConnect app under Settings > Product Info > Instant readout details > Encryption data > MAC Address."
```

"Product info" is sentence-cased in the Victron app:

```suggestion
          "address": "This Bluetooth address is automatically discovered. You may view a device's Bluetooth address in the VictronConnect app under Settings > Product info > Instant readout details > Encryption data > MAC Address."
```